app/views/schools/add_participants/_ect_or_mentor_answers.html.erb

```diff
@@ -73,12 +75,12 @@
     <% end %>
   <% end %>
 
-  <% if @school_cohort.default_induction_programme && @school_cohort.default_induction_programme.partnership&.active? %>
+  <% if form.show_default_induction_programme_details? %>
     <% summary_list.row do |row| %>
       <% row.key { "Training with" } %>
       <% row.value do %>
-        <p class="govuk-body"><%= @school_cohort.default_induction_programme.lead_provider&.name %></p>
-        <p class="govuk-body"><%= @school_cohort.default_induction_programme.delivery_partner&.name %></p>
+        <p class="govuk-body"><%= form.lead_provider&.name %></p>
+        <p class="govuk-body"><%= form.delivery_partner&.name %></p>
```

Comment:
Do we want empty paragraph tags here when the lead provider or delivery partner is unset? I think that might be an grey area on the accessibility front.

Maybe:

```suggestion
        <%= tag.p(form.lead_provider, class: "govuk-body") if form.lead_provider.present? %>
        <%= tag.p(form.delivery_partner, class: "govuk-body") if form.delivery_partner.present? %>
```